docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,105 @@
+
+2. Select your Namespace from the drop-down list and go to the **Catalog** view. See that your services are now available under the **Services** tab. Provision the service instance by choosing your service and clicking the **Add once** button in the top-right corner of the page.
+
+3. Create a lambda function. In the **Overview** tab, click the **Deploy new resource** button and upload the `lambda.yaml` file.
+
+4. Expose your lambda:
+
+    a. Go to the **Lambdas** tab in the left navigation panel and choose the `call-order-service` lambda.
+
+    b. In the **Settings & Code** section, click the **Select Function Trigger** button and expose your lambda via HTTPS.
+
+    c. Untick the **Enable authentication** field as  there is no need to secure the connection for the purpose of this tutorial.
+
+    d. Click **Add**.
+
+    e. Scroll down to the end of your lambda view and bind your lambda to your instance by clicking the **Create Service Binding** button in the **Service Binding** section. Choose the ServiceInstance you want to bind your lambda to and click **Create Service Binding**.
+
+    f. Save the settings in the right top right corner of the page.
+
+    g. Click the **Lambdas** tab and wait until the lambda status is completed and marked as `1/1`.
+
+5. Test your lambda. Navigate to your lambda and go to the **Testing** tab. After you click the **Send** button, you can see the following output in the **Response** field:
+
+    ```
+    {
+      "status": 200,
+      "data": []
+    }
+    ```
+
+    You can test your lambda by performing the following actions in the **Payload** field:
+
+    - `{"action":"add"}` - adds the new order
+    - `{"action":"list"}` - lists all orders; this is the default command executed after you click the **Send** button
+    - `{"action":"delete"}` - deletes all the orders
+
+### Cleanup
+
+Clean up your cluster after going through this tutorial. To do so, delete your resources in the following order:
+
+1. Go to the **Lambdas** tab, unfold the vertical option menu and delete your lambda.
+
+2. Go to the **Services** tab and delete `order-service`.
+
+3. Go to the **Deployments** tab and delete the `order-service` deployment.
+
+4. Go to the **APIs** tab and delete the `order-service` API.
+
+5. Go to the **Instances** tab, navigate to **Services**, and deprovision your instance by selecting the trash bin icon.
+
+6. Go to the **Overview** section and unbind `test-app` from your Namespace.
+
+7. Go back to the Namespaces view and delete your Namespace.
+
+8. In the Compass UI, remove `test-app` from the **Applications** view. If you go back to the **Applications** view in the Kyma Console UI, you can see that the `test-app` Application is removed. It can take a moment, refresh if the Application is still there.
```

Comment:
```suggestion
8. In the Compass UI, remove `test-app` from the **Applications** view. If you go back to the **Applications** view in the Kyma Console UI, you can see that the `test-app` Application is removed. It can take a moment, so refresh the Console UI if the Application is still there.
```